### ISSUE TITLE
extend news with an event function

### ISSUE THREAD

**Issue by rallek** [zikula-modules/News#132]
It is a feature request.

I do have the news module in use on all my sites since very long time. In parallel I do have a calender module (timeIt or newer Postcalendar). My users are acting different than expected. they want to have both in one. For a lot of dates they are creating in parallel a news as well.

What would be perfect? The implementation of some additional fileds as a kind of meta data:

Add a boolean field "isEvent". If this is selected in edit.tpl some more fields get visible. e.g. DateTime, Duration, Location (Fieldset should be discussed). News still do have only one table.

Additional create a block (and content plugin) with a list of upcomming events. Might be a view_event.tpl is also a good idea. The possibilities to filter this events should be similar to pure news.

This feature is usefull for communities like a sports club, our church and other similar. With this feature my sites would not need a Calendar anymore :-)

**Comment by Guite** [comment]
Closing as won't fix since we agreed that an event function should be handled by a different module which is included/attached to News articles using hooks.
